Assets/HoloToolkit/SpatialUnderstanding/Scripts/SpatialUnderstandingCustomMesh.cs

```diff
@@ -33,6 +33,11 @@ private float MaxFrameTimeInSeconds
             get { return (MaxFrameTime / 1000); }
         }
 
+        /// <summary>
+        ///  Whether to create mesh colliders. If unchecked, mesh colliders will be empty and disabled.
+        /// </summary>
+        public bool CreateMeshColliders = true;
```

Comment:
The only comment I have at this point is that I like the idea of this being a property following the behavior of `DrawProcessedMesh` such that setting it to false would disable the `Collider` for all the existing `surfaceObjects`